WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3565,17 +3567,18 @@ e_byzantium = {
 		capital = 560 # Cherson
 		
 		hellenic_pagan = 600
-		bosporan_faith = 200
+		bosporan_faith = 1000
```

Comment:
Hellenic Pagans have six hundred Crusade points on a title thatm eans nothing to their religion. The kingdom means EVERYTHING to the Bosporans, as it is the BIRTHPLACE of their religion. I will lower the Crusade points to the same as the Hellenic, since that seems to be your maximum.